package.json

```diff
@@ -47,6 +47,9 @@
   },
   "dependencies": {
     "joi": "^13.7.0",
-    "cron-parser": "^2.7.3"
+    "cron-parser": "^2.7.3",
+    "pg": "7.12.1",
```

Comment:
Where is `pg` used?

Reply:
it will be used as part of deployment. we use data-schema package alone for migration.

Reply:
And would you add `mysql2` same as [datastore-sequelize(]https://github.com/screwdriver-cd/datastore-sequelize/blob/6b84a91b0fbb2375009f796490f647f23a7befcf/package.json#L51)? I think it's also needed for mysql users.

Reply:
updated devDependencies with mysql2 package and readme.md file